weather_mv/loader_pipeline/util_test.py

```diff
@@ -42,4 +43,17 @@ def test_no_duplicate_coordinates(self):
 
 class ToJsonSerializableTypeTests(unittest.TestCase):
     # TODO(#106): Write tests...
-    pass
+    def assert_serializes_json(self, input):
```

Comment:
Please add a test case for every expected input type (check out the if statements in the method you're testing to infer which types to test). In general, every conditional block in the method should be exercised.

Reply:
yes, was a bit confused about the wiring so didn't make it through...looking forward to test it for all possible cases now